nautobot_ssot/integrations/infoblox/diffsync/adapters/infoblox.py

```diff
@@ -45,13 +45,56 @@ def __init__(self, *args, job=None, sync=None, conn=None, **kwargs):
             )
             raise PluginImproperlyConfigured
 
+    # credit to @Eric-Jckson for the conainer code addition: https://github.com/nautobot/nautobot-plugin-ssot-infoblox/pull/143
     def load_prefixes(self):
         """Load InfobloxNetwork DiffSync model."""
-        if PLUGIN_CFG.get("import_subnets"):
+
+        def get_tree_from_container(root_container: str) -> list:
+            """Returns the list of all child containers from a given root container."""
+            flattened_tree = []
+            stack = []
+            root_containers = self.conn.get_network_containers(prefix=root_container)
+            if root_containers:
+                stack = [root_containers[0]]
+
+            while stack:
+                current_node = stack.pop()
+                flattened_tree.append(current_node)
+                children = self.conn.get_child_network_containers(prefix=current_node["network"])
+                stack.extend(children)
+
+            return flattened_tree
```

Comment:
Can this be a normal method instead of a nested function?

Reply:
Done